include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -211,6 +212,14 @@ class SFML_AUDIO_API InputSoundFile
     void close();
 
 private:
+    ////////////////////////////////////////////////////////////
+    /// \brief Default constructor
+    ///
+    /// Useful for implementing close()
+    ///
+    ////////////////////////////////////////////////////////////
+    InputSoundFile() = default;
```

Comment:
I'm really not a fan of this but I can try to improve it in a separate PR.

Reply:
I'm open to better alternatives for implementing `close()` that don't require having to list all data members.